client/src/app/menu/menu.component.ts

```diff
@@ -30,72 +29,72 @@ export class MenuComponent implements OnDestroy, OnInit {
                 {
                     name: 'Tableau de bord',
                     route: '/dashboard',
-                    permissions: []
-                }
-            ]
+                    permissions: [],
+                },
+            ],
         },
         {
             links: [
                 {
                     name: 'Adhérents',
                     route: '/members',
-                    permissions: ['member:read']
-                }
-            ]
+                    permissions: ['member:read'],
+                },
+            ],
         },
         {
             links: [
                 {
                     name: 'Barmen',
                     route: '/barmen',
-                    permissions: ['barman:read']
-                }
-            ]
+                    permissions: ['barman:read'],
+                },
+            ],
         },
         {
             links: [
                 {
                     name: 'Kommissions',
                     route: '/kommissions',
-                    permissions: ['kommission:read']
-                }
-            ]
+                    permissions: ['kommission:read'],
+                },
+            ],
         },
         {
             links: [
                 {
                     name: 'Roles',
                     route: '/roles',
-                    permissions: ['roles:read']
-                }
-            ]
+                    permissions: ['roles:read'],
+                },
+            ],
         },
         {
             links: [
                 {
                     name: 'Comptes spéciaux',
                     route: '/specialaccounts',
-                    permissions: ['specialaccount:read']
-                }
-            ]
+                    permissions: ['specialaccount:read'],
+                },
+            ],
         },
         {
             links: [
                 {
                     name: 'Ouvrir les services',
                     route: '/open-services',
-                    permissions: ['service:write']
-                }
-            ]
-        }
+                    permissions: ['service:write'],
+                },
+            ],
+        },
     ];
 
     mobileQuery: MediaQueryList;
     sidenavQuery: MediaQueryList;
     user: ConnectedUser;
 
-    private _mobileQueryListener: () => void;
-    private _sidebavQueryListener: () => void;
+    _mobileQueryListener: () => void;
+    _sidebavQueryListener: () => void;
```

Comment:
Why did you remove the private keyword ?

Reply:
False positive TSLint error...